tests/testthat/test-tmb.R

```diff
@@ -711,6 +711,16 @@ test_that("h_mmrm_tmb_check_conv warns if convergence code signals non-convergen
   )
 })
 
+# h_get_x_matrix -----
+
+test_that("h_get_x_matrix(object, newdata) works", {
+  formula <- FEV1 ~ SEX + ARMCD * AVISIT + us(AVISIT | USUBJID)
+  result <- mmrm(formula, fev_data[1:100, ], reml = FALSE)
+  newdata <- fev_data[101:nrow(fev_data), ]
+  mmrm:::h_get_x_matrix(result, newdata)
+  mmrm:::predict.mmrm_tmb(result, newdata)
```

Comment:
you don't need the `mmrm:::`, also internal functions are available in tests

Reply:
helps with interactive testing, can remove later